pages/linux/tcpkill.md

```diff
@@ -0,0 +1,7 @@
+# tcpkill
+
+> Kills specified in-progress TCP connections.
+
+- Listen and select eth1 interface, host 192.168.4.27 and port 6666's connections to kill:
+
+`tcpkill -i eth1 host 192.168.4.27 port 6666`
```

Comment:
interface, host and port inside tokens (`{{}}`)

Reply:
@agnivade I see some old docs use specific example . Is that legacy?